### ISSUE TITLE
SecItemCopyMatching failed for key

### ISSUE THREAD

**Issue by magin** [granoff/Lockbox#27]
Somewhy I get this in my console when using Lockbox:

+[Lockbox objectForKey:] [Line 110] SecItemCopyMatching failed for key com.magin.MaginSpeedMode.udidKey2: -25308
+[Lockbox setObject:forKey:accessibility:] [Line 94] SecItemAdd failed for key com.magin.MaginSpeedMode.udidKey2: -25308

**Comment by magin** [comment]
Just to update: I am still using on daily basis `kSecAttrAccessibleAlwaysThisDeviceOnly` plus caching what Lockbox returns and had not had any problems since August.

**Comment by colasjojo** [comment]
The problem occurs here, with `iOS9`

```
-[Lockbox objectForKey:] [Line 133] SecItemCopyMatching failed for key com.company.app.AuthToken: -34018
```
